### DISCUSSION TITLE
Hocam Bitti ?

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#1826] [Soru Cevap - Türkçe] [answered]
Çok adaletsiz değilmi ya Herşey çok adaletsiz çekilişten dağıtılan ıbax lar vs.

![Aesthetic-Wallpaper](https://user-images.githubusercontent.com/98268472/152142208-30e351b2-2cf5-46f5-88fa-d7895b9c997d.jpg)

**Comment by Skips1** [comment]
Bekleniyor bayağıdır

**Comment by enzifiri** [comment, marked as answer]
Bekliyoruz bakalım.

**Reply by Skips1** [reply]
Goooooooooooooooooooooooooooooooooooddddddddddddd

**Comment by volky1983** [comment]
@ @ @ @ @

**Comment by enzifiri** [comment]
Bu neydi ya :D

**Reply by Skips1** [reply]
![image](https://github.com/IBAX-io/go-ibax/assets/98268472/ccc972c7-88bf-4226-925b-c4434ee8970f)
![image](https://github.com/IBAX-io/go-ibax/assets/98268472/ac99853f-6c9f-41fe-b9e5-b2a49ef4d9c5)
Hatırladın mı bişiler :D

**Comment by Enggun333** [comment]
Ben uzun vadede projeden umutluyum
